emojivoto-web-app/js/setup_dev_env.sh

```diff
@@ -150,19 +150,30 @@ connect_to_k8s() {
     send_telemetry "connectedToK8S"    
 }
 
-install_telepresence() {
+install_upgrade_telepresence() {
     display_step 3
-    echo 'Checking for Telepresence'
+    install_telepresence=false
+    echo -n 'Checking for Telepresence ... '
     _=$(which telepresence)
     if [ "$?" = "1" ]; then
+        install_telepresence=true
         echo "Installing Telepresence"
+    else
+        if telepresence version|grep upgrade >/dev/null 2>&1; then
+            install_telepresence=true
+            # daemon and client need to have same version
+            _=$(telepresence quit)
+            echo "Upgrading Telepresence"
```

Comment:
Not sure if we should ask the user before upgrading

Reply:
I think that is okay to upgrade without asking, otherwise, the script could fail. I believe that if they had a previous version is because they already interacted with telepresence and they would know how to downgrade if they need it. Also, the target of this script is first-time users, so I think that upgrading the CLI without asking is a good move.